devtools/server/actors/replay/telemetry.js

```diff
@@ -8,6 +8,16 @@ function pingTelemetry(source, name, data) {
 
   if (!enabled || !url) return;
 
+  const userInfo = Services.prefs.getStringPref("devtools.recordreplay.user");
```

Comment:
This pref no longer exists, it was replaced by `devtools.recordreplay.user-token`. We'll have to use that and parse it like https://github.com/RecordReplay/gecko-dev/blob/56e29838b8f37345ff0c411b2bce8db16bf5195a/devtools/server/actors/replay/connection.js#L328 does.

Also if the browser is using an API key, we'll want to skip using this pref.